cypress/support/Projects.js

```diff
@@ -4,647 +4,750 @@ const DEFAULT_TIMEOUT = 60000;
+    .should('exist')
+    .click();
+
+  // Wait for logout icon and force click
+  cy.get('[data-testid="logout-btn"]').should('exist').click({ force: true });
+
+  // wait for the request to complete
+  cy.wait('@logout', { timeout: DEFAULT_TIMEOUT });
+
+  // Verify logout by checking the welcome message and URL
+  cy.contains('Bike4Mind', { timeout: DEFAULT_TIMEOUT }).should('exist');
+  cy.url().should('contain', '/login');
 };
 
-
-
-const checkInbox = (projectName) => {
-    // Click the menu button
-    cy.get('[data-testid="notebook-sidenav-footer-menu-button"]', { timeout: DEFAULT_TIMEOUT })
-        .should('be.visible')
-        .click();
-
-    // Click inbox option
-    cy.contains('Inbox', { timeout: DEFAULT_TIMEOUT })
-        .should('be.visible')
-        .click();
+const checkInbox = projectName => {
+  // Click the menu button
+  cy.get('[data-testid="notebook-sidenav-footer-menu-button"]', {
+    timeout: DEFAULT_TIMEOUT,
+  })
+    .should('be.visible')
+    .click();
+
+  // Click inbox option
+  cy.contains('Inbox', { timeout: DEFAULT_TIMEOUT })
+    .should('be.visible')
+    .click();
 };
 
 const handleProjectInvite = (projectName, action) => {
-    // Click invites tab
-    cy.contains('Invites', { timeout: DEFAULT_TIMEOUT })
-        .should('be.visible')
-        .click();
-
-    // Validate project invitation
-    cy.contains(projectName, { timeout: DEFAULT_TIMEOUT })
-        .should('be.visible')
-        .click();
-
-
-    if (action === 'accept') {
-        // Click accept button
-        cy.get('[data-testid="CheckIcon"]', { timeout: DEFAULT_TIMEOUT })
-            .eq(0)
-            .should('be.visible')
-            .click();
-
-        // Verify success message
-        cy.contains('Successfully joined the project', { timeout: DEFAULT_TIMEOUT, matchCase: false })
-            .should('be.visible');
-
-    } else {
-        // Click deny button
-        cy.get('[data-testid="DoDisturbIcon"]', { timeout: DEFAULT_TIMEOUT })
-            .eq(0)
-            .should('be.visible')
-            .click();
-
-        // Verify denial message
-        cy.contains('Successfully refused the project', { timeout: DEFAULT_TIMEOUT, matchCase: false })
-            .should('be.visible');
-    }
-
-    // Close the modal
-    cy.get('.MuiModalClose-sizeMd', { timeout: DEFAULT_TIMEOUT })
-        .should('exist')
-        .click({ force: true });
+  // Click invites tab
+  cy.contains('Invites', { timeout: DEFAULT_TIMEOUT })
+    .should('be.visible')
+    .click();
+
+  // Validate project invitation
+  cy.contains(projectName, { timeout: DEFAULT_TIMEOUT })
+    .should('exist')
+    .click();
+
+  if (action === 'accept') {
+    cy.intercept('POST', '/api/invites/**/accept').as('inviteAccept');
+
+    // Click accept button
+    cy.get('[data-testid="invite-accept-btn"]', { timeout: DEFAULT_TIMEOUT })
+      .eq(0)
+      .should('be.visible')
+      .click();
+
+    cy.wait('@inviteAccept', { timeout: DEFAULT_TIMEOUT });
```

Comment:
Here's the specific fix for Invite Accept. The data-testid was invalid.
And also ensure the API completes before proceeding to checking for the success toaster